### PR TITLE
games-strategy/julius: version bump 1.6.0

### PR DESCRIPTION
**Pull request by TheChymera**
Package-Manager: Portage-3.0.20, Repoman-3.0.3
Signed-off-by: Horea Christian <chr@chymera.eu>

### CODE REVIEW

File: games-strategy/julius/julius-1.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+pkg_postinst() {
+	xdg_pkg_postinst
+
+	ewarn "Julius requires you to download the original Caesar 3 resources."
+	ewarn
+	ewarn "You need to obtain these files from a vendor of the proprietary software."
+	ewarn "You will then need to copy the 'app' directory into '~/.cache/julius/'."
+	ewarn "One way to obtain this directory is to download the GOG Cesar 3 edition."
+	ewarn "You can then produce this directory by running:"
+	ewarn "    innoextract -m setup_caesar3_2.0.0.9.exe"
```
Comment (ionenwks) [inline games-strategy/julius/julius-1.6.0.ebuild:44]: Be nice to mention where the command is from (full package name with category), even if it seem obvious someone may not necessarily try to just emerge the command's name.

File: games-strategy/julius/julius-1.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+DESCRIPTION="Feature-faithful open source re-implementation of Caesar III"
+HOMEPAGE="https://github.com/bvschaik/julius"
+SRC_URI="https://github.com/bvschaik/julius/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+
+DEPEND="
+	media-libs/libpng:0=
```
Comment (ionenwks) [inline games-strategy/julius/julius-1.6.0.ebuild:17]: Can drop the 0 (keep :=), old slots are long gone so not really necessary.

**Review by ionenwks — COMMENTED** [review]
Maybe a few small ebuild improvements?

**Comment by TheChymera** [comment]
@ionenwks done :)

**Comment by TheChymera** [comment]
@ionenwks ok, done.

**Review by ionenwks — APPROVED** [review]

**Pull request closed without merging**